### PR TITLE
Operation Bagnacavallo

### PR DESCRIPTION
**Pull request by sebbalex**
Following new PAs process described here: #334 

- PAs pages are autogenerated from crawler data, layout improvements
- Sort by filter in all search pages were not managed by l10n, now it is.
- PAs software page generation, layout and scheme
- Page for administrations list, layout and language switch feature enabled
- New URL location for PAs
- ITA translation of 'Read More'
- Lang Switch supports also administrations pages
- Switched back to /en/pa/* for PAs
- Lang switch now works with PAs
- Search page now has correct translation for filters

This PR needs new requirements on developers-italia-backend side to be merged (issue track will follow).

### CODE REVIEW

File: _data/l10n.yml [hunk 1]
```diff
@@ -197,6 +200,13 @@ it:
 
         * **Longevity**: l'età del progetto. I range di ogni misura possono essere trovati nel file *vitality-ranges.yml* in [https://github.com/italia/developers-italia-backend](https://github.com/italia/developers-italia-backend)"
 
+    pa:
+      top_right_text: Questa pagina elenca il software di cui l'ente è titolare e che è stato pubblicato con licenza aperta ai sensi dell'art. 69 del Codice dell'Amministrazione Digitale.
+      title: AMMINISTRAZIONI
+      more_details: Esplora
+      reuse_sw: SOFTWARE A RIUSO
+      third_party_sw: SOFTWARE OPEN-SOURCE DI TERZE PARTI
```
Comment (libremente) [inline _data/l10n.yml]:
```suggestion
      third_party_sw: SOFTWARE OPEN SOURCE DI TERZE PARTI
```

File: _data/l10n.yml [hunk 2]
```diff
@@ -393,5 +406,12 @@ en:
         * **User community**: the number of unique authors;
 
         * **Longevity**: the age of the project. The ranges of every measure can be found in the vitality-ranges.yml file in [https://github.com/italia/developers-italia-backend](https://github.com/italia/developers-italia-backend)"
+    pa:
+      top_right_text: This page lists the software solutions owned by this public entity and released under an open license according to art. 69 of the Digital Administration Code.
+      title: ADMINISTRATIONS
+      more_details: Read more
+      reuse_sw: REUSE SOFTWARE
+      third_party_sw: THIRD PARTY OPEN-SOURCE SOFTWARE
```
Comment (libremente) [inline _data/l10n.yml]:
```suggestion
      third_party_sw: THIRD PARTY OPEN SOURCE SOFTWARE
```

File: _data/l10n.yml [hunk 3]
```diff
@@ -393,5 +406,12 @@ en:
         * **User community**: the number of unique authors;
 
         * **Longevity**: the age of the project. The ranges of every measure can be found in the vitality-ranges.yml file in [https://github.com/italia/developers-italia-backend](https://github.com/italia/developers-italia-backend)"
+    pa:
+      top_right_text: This page lists the software solutions owned by this public entity and released under an open license according to art. 69 of the Digital Administration Code.
```
Comment (libremente) [inline _data/l10n.yml]:
```suggestion
      top_right_text: This page lists the software solutions owned by this public entity and released under an open license according to art. 69 of the Italian Digital Administration Code.
```
potrebbe aver senso aggiungere `Italian`?

File: _layouts/default-pa.html [hunk 4]
```diff
@@ -7,10 +7,59 @@
 
 <div class="u-background-95">
   <div id="main" class="u-layout-wide u-layoutCenter">
-    {% include header-pa.html hasSearch=true %}
+    {% include header.html hasSearch=true %}
+    {% include offcanvas.html menu_items=site.data.menu_items %}
+    {% comment %}
+    <!-- NOT FOR NOW-->
```
Comment (libremente) [inline _layouts/default-pa.html]: Che significa il commento?

File: _layouts/default-pa.html [hunk 5]
```diff
@@ -7,10 +7,59 @@
...
+    {% endcomment %}
+
+
+    {% if include.hasSearch %}
+    <div class="Header-search ml-4 mb-0" id="header-search">
+      {% include search_form.html textColor="primary" %}
+    </div>
+    {% endif %}
+
+
```
Comment (libremente) [inline _layouts/default-pa.html]: Eliminerei i due return.

File: _layouts/software-subcat-pa.html [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+  </div>
+</div>
+<div class="container page-softwares-list">
+  <div class="row">
+    <div class="col-md-12">
+      <div class="intro">
+        <h1 style="font-size: 3rem">{{ page.title }}</h1>
+        <div class="abstract-sorting row row-eq-height d-none d-md-flex">
+          <div class="results-number col-md-6 d-flex"></div>
+
```
Comment (libremente) [inline _layouts/software-subcat-pa.html]: Toglierei questo return.

**Comment by alranel** [comment]
Depends on https://github.com/italia/developers-italia-backend/issues/91

File: _data/l10n.yml [hunk 7]
```diff
@@ -197,6 +200,13 @@ it:
 
         * **Longevity**: l'età del progetto. I range di ogni misura possono essere trovati nel file *vitality-ranges.yml* in [https://github.com/italia/developers-italia-backend](https://github.com/italia/developers-italia-backend)"
 
+    pa:
+      top_right_text: Questa pagina elenca il software di cui l'ente è titolare e che è stato pubblicato con licenza aperta ai sensi dell'art. 69 del Codice dell'Amministrazione Digitale.
+      title: AMMINISTRAZIONI
+      more_details: Esplora
+      reuse_sw: SOFTWARE A RIUSO
```
Comment (alranel) [inline _data/l10n.yml]: Queste chiavi ci sono già: `software.software_open` e `software.reuse_software`. Conviene riutilizzarle, e renderle maiuscole via CSS

File: _layouts/default-pa.html [hunk 8]
```diff
@@ -7,10 +7,59 @@
...
+          <img src="{{ page.publicAuthority.emblem }}" alt="">
+          {% endif %}
+        </a>
+      </div>
+
+      <div class="Header-title col-md-7 col-9 mr-auto">
+        <h1 class="Header-titleLink Header-titleLink--pa">
+          <a class="text-900" href="{{ page.publicAuthority.website }}" target="_blank" rel="noopener noreferrer">
+            {% if page.publicAuthority.styledName %}
+            {{ page.publicAuthority.styledName }}
```
Comment (alranel) [inline _layouts/default-pa.html]: Nobody is ever going to provide a `styledName`... We could make this in the Jekyll template with a few heuristics, like stripping "Comune di", "Regione di", "Ministero del " etc.

**Comment by sebbalex** [comment]
It also fixes the #338

File: _layouts/default-pa.html (hunk 4 above)
Comment (sebbalex) [reply _layouts/default-pa.html]: È un qualcosa di legacy, va contestualizzato

**Comment by sebbalex** [comment]
https://github.com/italia/developers-italia-backend/issues/91

Issue related

File: _data/l10n.yml (hunk 7 above)
Comment (sebbalex) [reply _data/l10n.yml]: ok, utilizzato `upcase` di Jekyll

File: _pages/it/bagnacavallo.md [hunk 9]
```diff
@@ -13,4 +13,4 @@ publicAuthority:
     websiteText: www.comune.bagnacavallo.ra.it
 ---
 
-ciao
\ No newline at end of file
+ciao
```
Comment (libremente) [inline _pages/it/bagnacavallo.md]: Ciao!
Mi sa che questa è da cancellare :)

Comment (alranel) [reply _pages/it/bagnacavallo.md]: 👋

File: _data/l10n.yml [hunk 10]
```diff
@@ -335,6 +342,9 @@ en:
       software_open: Software open source
       reuse_software: Reuse software
       sort_by: Sort by
+      sort_by_relevance: Relevance
+      sort_by_emerging: Emerging
```
Comment (libremente) [inline _data/l10n.yml]: Riporto i miei dubbi qui (così li tracciamo). Hai scoperto che fa `emerging`? @sebbalex

Comment (sebbalex) [reply _data/l10n.yml]: Credo che fosse per ordinare i softwares nella ricerca per data di rilascio

**Comment by sebbalex** [comment]
#328

**Pull request merged**